bigbluebutton-tests/playwright/core/page.js

```diff
@@ -377,6 +377,16 @@ class Page {
     const ytFrame = new Page(this.page.browser, frame);
     return ytFrame;
   }
+
+  async hasElementChecked(selector, value, description) {
+    const locator = await this.page.locator(selector);
+    await expect(locator, description).toBeChecked(value);
+  }
+
+  async hasElementNotChecked(selector, value, description) {
+    const locator = await this.page.locator(selector);
+    await expect(locator, description).not.toBeChecked(value);
+  }
```

Comment:
`toBeChecked` does not receive any string parameter (double check [docs](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-checked))

```suggestion
  async hasElementChecked(selector, description) {
    const locator = await this.page.locator(selector);
    await expect(locator, description).toBeChecked();
  }

  async hasElementNotChecked(selector, description) {
    const locator = await this.page.locator(selector);
    await expect(locator, description).not.toBeChecked();
  }
```